####################################################################################################################
multiplier: 1   # Options: 1, 2, 3, 4

control_yaw: true
control_pitch: true
control_depth: true
control_distance: true
control_surge_speed: true
control_sway_speed: true

tolerance_depth: 0.1        #[m]
tolerance_distance: 0.05     #[m]
tolerance_min_distance: 0.20 #[m]
tolerance_max_distance: 1.0 #[m]
tolerance_yaw: 0            #[deg]
tolerance_pitch: 2          #[deg]

pitch_gain: 0.4
min_distance_gain: 0.08
max_distance_gain: 0.3

min_distance_gain_negative: 0.08

safety_timeout: 50.0   #[s]

######################################################################################################################
###############################################    MULTIPLIER TABLE    ###############################################
######################################################################################################################
#      VALUE     #     SURGE SPEED    #     SWAY SPEED    #     YAW RATE     #    DEPTH RATE    #     PITCH RATE     #
# ------------------------------------------------------------------------------------------------------------------ #
#       1        #      0.25 m/s      #      0.25 m/s     #     10 deg/s    #    0.125 m/s     #      5.0 deg/s     #
# ------------------------------------------------------------------------------------------------------------------ #
#       2        #      0.38 m/s      #      0.38 m/s     #     15 deg/s    #    0.187 m/s     #      7.5 deg/s     #
# ------------------------------------------------------------------------------------------------------------------ #
#       3        #      0.50 m/s      #      0.50 m/s     #     20 deg/s    #    0.250 m/s     #       10 deg/s     #
# ------------------------------------------------------------------------------------------------------------------ #
#       4        #      0.75 m/s      #      0.75 m/s     #     30 deg/s    #    0.375 m/s     #       15 deg/s     #
# ------------------------------------------------------------------------------------------------------------------ #